server/mongo.js

```diff
@@ -1,10 +1,11 @@
-//TODO: refactor here
-const mongoose = require('mongoose');
-let testModel = mongoose.model('users', (req,res)=>{
-    name:String
-})
-console.log('costam')
-mongoose.connect('mongodb://localhost/mydb', {useUnifiedTopology: true,useNewUrlParser: true});
-console.log('costam')
-
-testModel.findOne(function(error, result) { /* ... */ });
+import mongoose from 'mongoose';
+import User from './models/User.js';
+import Task from './models/Task.js';
+ 
+const connectDb = () => {
+  return mongoose.connect("mongodb://localhost:27017/myDb", {useUnifiedTopology: true});
```

Comment:
please store this hostname and port into config file

Reply:
PORT and DATABASE_URL were putted in the config file